packages/react-core/src/index.js

```diff
@@ -12,6 +12,7 @@ export { InvalidColumnError } from './utils/InvalidColumnError';
 export { debounce } from './utils/debounce';
 export { throttle } from './utils/throttle';
 export { randomString } from './utils/randomString';
+export { assert } from './utils/assert';
```

Comment:
This is internal API, not to be used by public, please export it with as`_assert`.

Then, use it like 

    import { _assert as assert } from '@carto/react-core/';

in other modules

Reply:
done